### PR TITLE
Non-rule-change: Names

### PR DESCRIPTION
**Pull request by pimotte**
Feel free to suggest better ones. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,76 +79,112 @@ proposal to amend or transmute it. If an amendment is amended or repealed, the
...
+
+Whatever is not prohibited or regulated by a rule is permitted and
 unregulated, with the sole exception of changing the rules, which is permitted
 only when a rule or set of rules explicitly or implicitly permits it.
 
 Mutable Rules 
 -------------
 
-**303** A player may have an infinite number of concurrent turns.
+**303** *Everybody Gets a Turn!*
```
Comment (MrHug) [inline README.md]: "a Turn"?

File: README.md [hunk 2]
```diff
@@ -63,76 +79,112 @@ proposal to amend or transmute it. If an amendment is amended or repealed, the
...
+One turn consists of proposing one rule-change and having it voted on.
 
-**203** A rule-change is adopted if and only if at least a simple majority
+**203** *Number of Required Ayes*
+
+A rule-change is adopted if and only if at least a simple majority
 of the players votes in favour of the rule-change.
 
-**204** An adopted rule-change takes full effect at the moment of the completion
+**204** *Moment of Effect*
```
Comment (MrHug) [inline README.md]: ==> "Effective Immediately"?

File: README.md [hunk 3]
```diff
@@ -214,19 +276,25 @@ to future warnings.
...
-**321** A vote can be retracted by commenting on the pull request with a comment that 
+**321** *Retraction Watch*
+
+A vote can be retracted by commenting on the pull request with a comment that 
 consists only of ":wave:". This will retract any votes made by the player that were made before
 this comment. A vote can only be retracted as long as it is valid and the rule-change is not
 yet adopted or rejected.
 
-**313** Any vote through a comment on a pull request is invalid, unless one of the
+**313** *日本語とトルキ語*
```
Comment (MrHug) [inline README.md]: @jdonkervliet do your part :D

**Comment by MrHug** [comment]
A few suggestions, mostly looks good. I would also be ok with the current names.

File: README.md [hunk 4]
```diff
@@ -150,22 +196,43 @@ of the rule.
...
+>>>>>>> origin/master
 gains 1 point.
 
-**315** Each eligible voter always has exactly one vote.
 
+**315** *Democracy Reloaded*
+
+Each eligible voter always has exactly one vote.
+
+<<<<<<< HEAD
```
Comment (MrHug) [inline README.md]: These things should go ;)

**Comment by pimotte** [comment]
Fixed. I blame Jesse.

**Comment by pimotte** [comment]
This is a kind of a pain to keep up to date, so I'd really appreciate an upvote here:)

**Comment by jdonkervliet** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by MrHug** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by pimotte** [comment]
大丈夫ではありません。= It's not okay.

Sorry:)

**Comment by jdonkervliet** [comment]
:+1:

**Comment by Jason-Statham** [comment]
More votes are needed!

**Comment by pimotte** [comment]
2/3 votes means merge:)

**Pull request merged**